data-raw/healthy-lives/behavioural-risk-factors/alcohol-misuse.R

```diff
@@ -0,0 +1,87 @@
+# Load packages
+library(tidyverse)
+library(readxl)
+library(sf)
+library(httr)
```

Comment:
- Please remove repetition of libraries
- dplyr is part of tidyverse and doesn't need to be separately loaded
- What did you use the sf package for?